<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseProfessionalActionDialog</class>
 <widget class="QDialog" name="chooseProfessionalActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Wybierz akcje</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="findProfessionalRadioButton">
       <property name="text">
        <string>Znajdź fachowca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="addProfessionalRadioButton">
       <property name="text">
        <string>Dodaj fachowca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="modifyProfessionalRadioButton">
       <property name="text">
        <string>Modyfikuj fachowca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="deleteProfessionalRadioButton">
       <property name="text">
        <string>Usuń fachowca</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkpushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
